apps/src/templates/teacherNavigation/ResourceRow.tsx

```diff
@@ -28,13 +28,17 @@ const ResourceRow: React.FC<ResourceRowProps> = ({
   const resourcePositionLabel = (
     <strong>{unitNumber + '.' + lessonNumber + ' '}</strong>
   );
+
+  const resourceDisplayText = () =>
+    resource.type ? `${resource.type}: ${resource.name}` : resource.name;
```

Comment:
do we plan to translate this page for launch? here is how to get the translated name of a resource: https://github.com/code-dot-org/code-dot-org/blob/ad90d0f431b63944da72a5ff3a2ad701d3b1aed3/dashboard/app/models/resource.rb#L85-L90

TBH I'm not sure if this kind of resource is exactly what you are showing here. if it is, let's show the translation now. if translations have not been set up for this new kind of resource, let's make sure we have a ticket to track that.

Reply:
<img width="1370" alt="image" src="https://github.com/user-attachments/assets/6f303ba3-22f9-4cbc-a346-33a596cf9e5c">
I think this is actually handled in the [summarize_for_lesson_materials](https://github.com/code-dot-org/code-dot-org/blob/369e970aa4d567ca0730a982be36f5387832db8d/dashboard/app/models/lesson.rb#L487) function (which calls, eventually, `summarize_for_lesson_plan` which uses this `get_localized_property`)

Reply:
ok, great! thanks for checking 👍